Source/WebCore/Modules/modern-media-controls/controls/macos-fullscreen-media-controls.js

```diff
@@ -175,12 +176,16 @@ class MacOSFullscreenMediaControls extends MediaControls
 
         this._lastDragPoint = this._pointForEvent(event);
 
-        this.element.addEventListener("mousemove", this, true);
```

Comment:
I think you need to keep this around, the event listeners have different behavior in capture and non-capture mode. Does anything go wrong if you keep this line and the matching `removeEventListener()` call?